src/signals/IncidentMap/components/FilterPanel/FilterCategory.tsx

```diff
@@ -22,7 +22,15 @@ export const FilterCategory = ({
       data-testid={text}
       id={text}
       checked={selected}
-      onChange={onToggleCategory}
+      onChange={() => {
+        onToggleCategory()
+      }}
```

Comment:
Why did you change this? It does the same `onChange={onToggleCategory}` right?

Reply:
oh yeah, will change it back.